### DISCUSSION TITLE
Is there a way to make a table or repeater for many-to-many relationship on create form?

### DISCUSSION THREAD

**Discussion by Demonmodez** [filamentphp/filament#1369] [Help] [answered]
Hi, I want to make a create form for Orders. In this case, Order has a many-to-many relationship with Product. On the Order's create form, I want to make like a repeater or table so I can add many Products to my Order before I actually save the Order. Is it possible to do it? Thank you very much..

**Comment by danharrin** [comment, marked as answer]
Hey, yes you could create a custom repeater similar to the `HasManyRepeater` that saves to a `BelongsToMany` relationship instead. `BelongsToManyRepeater` is on the roadmap so coming soon 👍

**Reply by Demonmodez** [reply]
Okay.. Thanks a lot 👍

**Reply by romaldyminaya** [reply]
@danharrin Hi Again,

I already implemented this in a proyect, here's a screenshot:

![image](https://user-images.githubusercontent.com/2809147/156386087-6f09c846-3f48-4de5-adbc-424db4ccd7de.png)

I called it HasManyTable. How can I PR this for you to merge it in the main proyect, is there any naming preffered for this component or something like that ?

**Reply by danharrin** [reply]
That looks cool, but I don't think I'd be willing to maintain it in the main project just yet. You could release it as a plugin yourself though, and if loads of people use it then we can take over?

**Reply by danharrin** [reply]
https://github.com/icetalker/filament-table-repeater

**Comment by danielbidala** [comment]
@Demonmodez For this kind of project I suggest to have an Orderitem model with it's own db table. You can do lot more with this pattern.

Order hasMany Orderitem <> Orderitem belongsTo Order
Product hasMany Orderitem <> Orderitem belongsTo Product

**Comment by romaldyminaya** [comment]
@Demonmodez did you manage to resolve that by yourself ? In that case would it be possible for you to share it with the community ?
